### PR TITLE
Fix model location and add Item properties

### PR DESCRIPTION
**Pull request by wisniewski-patryk**
This pull request fixes the location of the model file and adds the Length property to the Item model with a default value for CreatedDate.

### CODE REVIEW

**Review by wisniewski-patryk — COMMENTED** [review]
before you run localy, run docker with database and go to (in terminal) folder ./backend/FactoryOps.Api and run command `dotnet ef database update`

File: backend/FactoryOps.Api/Controllers/ItemsController.cs [hunk 1]
```diff
@@ -9,31 +10,42 @@ namespace FactoryOps.Api.Controllers;
...
-	public ActionResult<IAsyncEnumerable<Item>> GetWorkItems() => Ok(itemsRepository.GetAll());
-	
+	public ActionResult<IAsyncEnumerable<ItemDto>> GetAll()
+	{
+		return Ok(itemsRepository.GetAll()
+			.Select(item => new ItemDto(
+				item.Id,
+				item.Group,
+				item.Title,
+				item.StartTime.ToString("yyyy-MM-dd:HH:mm:ss"),
```
Comment (rymarczyk-tomasz) [inline backend/FactoryOps.Api/Controllers/ItemsController.cs:20]: I don't think we need to use seconds when we have elements moved every hour

File: backend/FactoryOps.Api/Controllers/ProgrammersController.cs
Comment (rymarczyk-tomasz) [inline backend/FactoryOps.Api/Controllers/ProgrammersController.cs:1]: Will it fetch developers from the database?

File: .vscode/settings.json [hunk 2]
```diff
@@ -11,5 +11,9 @@
 		"editor.defaultFormatter": "esbenp.prettier-vscode"
 	},
 
-	"dotnet.defaultSolution": "backend/FactoryOps.sln"
+	"dotnet.defaultSolution": "backend/FactoryOps.sln",
+	"cSpell.words": [
+		"HSTM",
+		"Manzanillo"
```
Comment (rymarczyk-tomasz) [inline .vscode/settings.json:17]: And what is it? Why is manzanillo served?

File: backend/FactoryOps.Api/Database/Models/CreateItemModel.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+namespace FactoryOps.Api.Database.Models;
+
+public record CreateItemModel(
+	int Group,
+	string Title,
+	string StartTime,
+	int Length,
```
Comment (rymarczyk-tomasz) [inline backend/FactoryOps.Api/Database/Models/CreateItemModel.cs:7]: Maybe let's change this variable to hours, or at least lengthInHours

File: .vscode/settings.json (hunk 2 above)
Comment (wisniewski-patryk) [reply .vscode/settings.json:17]: it's for checking spelling

File: backend/FactoryOps.Api/Controllers/ProgrammersController.cs
Comment (wisniewski-patryk) [reply backend/FactoryOps.Api/Controllers/ProgrammersController.cs:1]: what developers?

Comment (rymarczyk-tomasz) [reply backend/FactoryOps.Api/Controllers/ProgrammersController.cs:1]: programmers*

File: backend/FactoryOps.Api/Controllers/ItemsController.cs (hunk 1 above)
Comment (wisniewski-patryk) [reply backend/FactoryOps.Api/Controllers/ItemsController.cs:20]: I think we won't use even minutes and hours, but for now, until we receive any feedback, I want to save it.

Comment (rymarczyk-tomasz) [reply backend/FactoryOps.Api/Controllers/ItemsController.cs:20]: ok

File: backend/FactoryOps.Api/Controllers/ProgrammersController.cs
Comment (wisniewski-patryk) [reply backend/FactoryOps.Api/Controllers/ProgrammersController.cs:1]: ohh.. yeah, now we have programmers in the database and can store which item is being made by a particular programmer.

File: backend/FactoryOps.Api/Database/Models/CreateItemModel.cs (hunk 3 above)
Comment (wisniewski-patryk) [reply backend/FactoryOps.Api/Database/Models/CreateItemModel.cs:7]: great idea <3

**Review by rymarczyk-tomasz — APPROVED** [review]

**Pull request merged**